test/unit/lifecycle/properties.js

```diff
@@ -325,4 +338,21 @@ describe('lifecycle/property', function () {
       });
     });
   });
+  
+  describe('patterns', function () {
```

Comment:
This was the actual bug where you wanted to use the internal value in the setter when you specify a custom getter.